src/unix/aix-common.c

```diff
@@ -234,8 +248,7 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
 
     memcpy(flg.ifr_name, p->ifr_name, sizeof(flg.ifr_name));
     if (ioctl(sockfd, SIOCGIFFLAGS, &flg) == -1) {
-      uv__close(sockfd);
-      return UV_ENOSYS;
+      goto syserror;
     }
```

Comment:
Style: no braces around single-line consequents. Happens in a few other places below.

Reply:
fixed